theme/static/js/modal.js

```diff
@@ -0,0 +1,85 @@
+Vue.component('modal', {
```

Comment:
Clean up comments and console logs in this file. Also consider moving into the hs-vue directory with the rest of the Vue component scripts.

Reply:
Actually, should it be in modals.js in the same directory?

Reply:
If this component is meant to be reused I'd rather have it inside the hs-vue directory, otherwise in models.js

Reply:
Won't that directory eventually be too large? If anything, I'd like to start moving away from a colossal frontend-themed Django app, in `theme`. Let me know your final preference: 1. `hs_communities` app 2. hs-vue directory in `theme` 3. and I'm still wondering about `modals.js` file and eventually reducing sprawl across all the modals everywhere, across templates and apps